### ISSUE TITLE
cannot add css library Bootstrap

### ISSUE THREAD

**Issue by syddo** [electron-react-boilerplate/electron-react-boilerplate#2486]
## Prerequisites

- [x] Using yarn
- [x] Using node 10.x
- [x] Using an up-to-date [`master` branch](https://github.com/electron-react-boilerplate/electron-react-boilerplate/tree/master)
- [x] Using latest version of devtools. See [wiki for howto update](https://github.com/electron-react-boilerplate/electron-react-boilerplate/wiki/DevTools)
- [x] Link to stacktrace in a Gist (for bugs) [link to Gist](https://gist.github.com/syddo/66a9c6075f2aa74c2c50cd7073995ee1.js)
- [ ] For issue in production release, devtools output of `DEBUG_PROD=true yarn build && yarn start`
- [x] Tried solutions mentioned in [#400](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/400)

## Expected Behavior
no errors during package install via yarn add

## Current Behavior
errors during package install

## Steps to Reproduce (for bugs)

```
$ git clone --depth=1 https://github.com/chentsulin/electron-react-boilerplate.git your-project-name
$ cd your-project-name
$ yarn
$ yarn add react-bootstrap bootstrap
```

## Context
In my app, I would like to use React Bootstrap

## Your Environment

- Node version :v12.16.3
- Version or Branch used :master
- Operating System and version : macOS Catalina
- Link to your project :na

**Comment by amilajack** [comment]
Please share stacktrace

**Comment by amilajack** [comment]
Fixed in #2465

**Comment by songyuc** [comment]
Hi, @syddo , do your bootstrap work normally on electron-react-boilerplate?
